protobuf/transaction.proto

```diff
@@ -219,5 +219,15 @@ message Transaction {
                 }
             }
         }
+
+        message Rule {
+            message Req {
+                string label = 1;
+                RuleMethod.Req method = 2;
+            }
+            message Res {
+                RuleMethod.Res response = 1;
+            }
```

Comment:
These newly introduced messages are not used anywhere. I presume they need to be used in `Transaction.Req` and `Transaction.Res` above?